monkey/infection_monkey/exploit/powershell_utils/powershell_client.py

```diff
@@ -42,14 +43,16 @@ def format_password(credentials: Credentials) -> Optional[str]:
     if credentials.secret_type == SecretType.CACHED:
         return None
 
+    plaintext_secret = get_plain_text(credentials.secret)
+
     if credentials.secret_type == SecretType.PASSWORD:
-        return credentials.secret
+        return plaintext_secret
 
     if credentials.secret_type == SecretType.LM_HASH:
-        return f"{credentials.secret}:00000000000000000000000000000000"
+        return f"{plaintext_secret}:00000000000000000000000000000000"
```

Comment:
For readibility, consider (in UTs as well):
```suggestion
        return f"{plaintext_secret}:{'0'*32}"
```